build_binaries.sh

```diff
@@ -5,26 +5,13 @@
 set -eEuo pipefail
 umask 022
 
-declare CONTAINER_ID
-cleanup() {
-    if [ -n ${CONTAINER_ID:-} ] ; then
-        docker stop $CONTAINER_ID \
-            || docker kill $CONTAINER_ID \
-            || true
-        docker rm $CONTAINER_ID \
-            || true
-    fi
-}
-trap cleanup EXIT HUP INT
-
 PROJECT=$(git rev-parse --short HEAD)
 echo "Building opendps git version: $PROJECT"
 
 docker rmi --no-prune opendps_$PROJECT || true
 docker build -t opendps_$PROJECT docker
 
-mkdir output
+mkdir output || true
```

Comment:
Would `mkdir -p output` be better? That way if there are any errors (besides if the directory already exists) they will show up in the log